docs/source/glossary.rst

```diff
@@ -0,0 +1,78 @@
+Glossary
+========
+
+.. _Umbral: https://github.com/nucypher/umbral-doc/blob/master/umbral-doc.pdf
+
+.. glossary::
+
+    Alice
+      *"The Data Owner"* :term:`Character` - retains full control over the data encrypted for her and determines whom to share the data with.
+
+    Bob
+      *"The Data Recipient"* :term:`Character` - the data recipient that :term:`Alice` intends to share data with.
+
+    Capsule
+      Encrypted symmetric key (:term:`KEM`) that is eventually re-encrypted.
+
+    cFrag
+      A fragment of ciphertext that is a partial re-encryption produced by a :term:`kFrag` operation on a :term:`Capsule`.
+
+    Character
+      A common term for any entity fulfilling a particular role in our cryptographic narrative.
+
+    DEM
+      Data encapsulation mechanism - data encrypted with a symmetric key.
+
+    Enrico
+      *"The Encryptor"* :term:`Character` - a data source that encrypts data on behalf of :term:`Alice` and produces a :term:`MessageKit`.
+
+    Felix
+      *"The Faucet"* :term:`Character` - provides *testnet* NU tokens for nodes on the test NuCypher Network.
+
+    KEM
+      Key encapsulation mechanism - a symmetric key encrypted with an asymmetric key
+
+    kFrag
+      A fragment of a :term:`Re-encryption Key`.
+
+    Label
+      A title for a classification/categorization of data according to how it is intended to be shared.
+
+    MessageKit
+      The :term:`KEM` and :term:`DEM` encrypted data that is stored together.
+
+    Moe
```

Comment:
Same question as for Felix.

Reply:
Less strong feeling about "Moe" but people will see it in the code... - at this point feels like why not